apps/design-system-docs/docs/components/packages/a8c-ui.mdx

```diff
@@ -0,0 +1,7 @@
+---
+sidebar_position: 2
```

Comment:
Is it intentional that `@automattic/ui` components is below `@wordpress/components`? I'm asking because it's not alphanumeric order and that may be counterintuitive.

Reply:
True, fixed in 707a4b0